CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ folder structure is present irrespective of using InnerEye as a submodule or not
 environment will be contained in the model.
 
 ### Removed
+- Removed blobxfer completely
```

Comment:
Can you say a bit more what consequences that has?
 Also say that two configuration options are no longer needed.

Reply:
... because people will see error messages if they feed old config file into new code